viam_cartographer.go

```diff
@@ -191,22 +199,36 @@ func New(
 	cancelCartoFacadeCtx, cancelCartoFacadeFunc := context.WithCancel(context.Background())
 
 	// use the override in testing if non nil
-	// otherwise use the lidar from deps as the
-	// timed sensor
-	timedLidar := testTimedSensorOverride
+	// otherwise use the sensors from deps as the
+	// timed sensors
+	timedLidar := testTimedLidarSensorOverride
+	timedIMU := testTimedIMUSensorOverride
 	if timedLidar == nil {
-		timedLidar = lidar
+		timedLidar = lidarObject
+	}
+	if timedIMU == nil {
+		timedIMU = imuObject
+	}
+
+	lidar := Lidar{
```

Comment:
Is there any better names than Lidar and imu you can think of for this? LiDARInfo? ViamLiDAR? LidarObject?

Reply:
Hmmmm... I think LidarObject should refer to the lidar that is created when calling sensors.NewLidar()... I like Lidar, as we can then call lidar.name, lidar.dataratems, etc...
But if you prefer having it called something else, then my preference would be LidarInfo.

Reply:
Lets leave it as is for now